cachi2/core/package_managers/gomod.py

```diff
@@ -974,9 +971,14 @@ def _fail_unless_allowed(module_name: str, package_name: str, allowed_patterns:
     is_submodule = _contains_package(versionless_module_name, package_name)
     if not is_submodule and not any(fnmatch.fnmatch(package_name, pat) for pat in allowed_patterns):
         raise UnsupportedFeature(
-            f"The module {module_name} is not allowed to replace {package_name} with a local "
-            f"dependency. Please contact the maintainers of this Cachito instance about adding "
-            "an exception."
+            reason=(
+                f"The module {module_name} is not allowed to replace {package_name} with a local "
+                "dependency."
+            ),
+            solution=(
+                "Please allow the replacement in Cachi2's configuration, or contact someone who "
+                "has access to the configuration."
+            ),
```

Comment:
We have a bit of a disconnect here between what Cachi2 is and how we know it will be used. Should we even mention that someone other than the user might be in charge of the configuration? Do we need something like a configurable "admin contact"?

Luckily this case might be irrelevant because we want to get rid of the allowlists

Reply:
For now, I'd go with:
```
Please allow the replacement in Cachi2's configuration, which by default lives at {path to config file}.
If you're using Cachi2 in a managed enviroment, please contact the administrators.
```